packages/app/src/components/applicationStages/ApplicationStages.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
+import nominationsAPI from '../../utils/API/nominationsAPI';
+import './style.css';
+
+const ApplicationStages = () => {
+  const [activeNomination, setActiveNomination] = useContext(
+    ActiveNominationContext
+  );
+  const [currentStatus, setCurrentStatus] = useState();
+  const status = [
+    'Received',
+    'Awaiting HIPAA',
+    'HIPAA Verified',
+    'Document Review',
+    'Ready for Board Review',
+  ];
+  useEffect(() => {
+    setCurrentStatus(capitalize(activeNomination.status));
+  }, [activeNomination, currentStatus]);
+  function createStatusEl() {
```

Comment:
Nitpick: Please add spaces between separate declarations (e.g. after the end of a function) for readability.